packages-urls/manager-install-rpm.yaml

```diff
@@ -0,0 +1,2 @@
+# TODO: Change ga-build to ga-release after nightly pipeline is operational
+http://cloudify-release-eu.s3.amazonaws.com/cloudify/17.11.20/build/cloudify-manager-install-17.11.20-community.x86_64.rpm
```

Comment:
the premium one did have a newline at the end, this one doesn't?! :P

Reply:
Good lord! This must be fixed right away!